pkg/util/os.go

```diff
@@ -14,3 +16,20 @@ func SleepAndReboot() error {
 	time.Sleep(sleepInterval)
 	return exec.Command("/usr/sbin/reboot", "-f").Run()
 }
+
+// Get disk size in bytes
+func GetDiskSizeBytes(devPath string) (uint, error) {
+	cmd := exec.Command("/usr/bin/lsblk", "--bytes", "--nodeps", "--raw", "--noheadings", "--output=SIZE", devPath)
+	bytes, err := cmd.Output()
+	if err != nil {
+		return 0, err
+	}
+
+	output := strings.TrimSpace(string(bytes))
+	sizeBytes, err := strconv.ParseUint(output, 10, 64)
+	if err != nil {
+		return 0, err
+	}
+
+	return uint(sizeBytes), nil
```

Comment:
prefer to return `unit64` instead of the machine-dependent type.

Reply:
Use `uint64` instead of `uint`.